encrypter.cpp

```diff
@@ -0,0 +1,168 @@
+#include <iostream>
+#include <fstream>
+#include <regex>
+#include <string>
+#include <stdlib.h>
+#include <unistd.h>
+
+using namespace std;
+
+string read_file(string& address, string error_message, bool clean = false)
+{
+  ifstream f;
+
+  f.open(address);
+  if(!f)
+  {
+    cout << error_message;
+    f.close();
+    exit(1);
+  }
+
+  char c;
+  string text = "";
+
+
+  while (f.get(c))
+    if ((c != ' ') && (c != '\n') && (c != '.') || clean)
+      text += c;
+
+  f.close();
+
+  if (text == "")
+  {
+    cout << "Given file is empty";
+    exit(1);
+  }
+
+  return text;
+}
+
+void record_file(string& address, string& text)
+{
+  ofstream f;
+  f.open(address);
+  f << text;
+  f.close();
+}
+
+string encrypt(string& text, string& key)
+{
+  int shift = rand() % 100;
```

Comment:
Do you now the range of rand() result? What if it will be as short type and You file is several GB size?
"Shift" here is a bad thing.